templates/generator/src/utils/conversions.ts

```diff
@@ -0,0 +1,76 @@
+import { ObjectMap, Operation, Parameter } from "@apexlang/core/model";
+import { convertType } from "./types";
+
+/**
+ * Convert a description to the appropriate format for the destination.
```

Comment:
Should we point users to this utility for the majority of use cases?

https://github.com/apexlang/codegen/blob/861cbd4738d533013c6587c371ee794e3cc8eff2/src/utils/utilities.ts#L266-L340

Reply:
Moved logic to formatComment